content/applications/marketing/events/event_tracks.rst

```diff
@@ -0,0 +1,207 @@
+create a new one from the *Events* application.
+
+To do that, navigate to :menuselection:`Events app`, and either select a pre-existing event from the
+:guilabel:`Events` dashboard, or create a new one by clicking :guilabel:`New`.
+
+Once on the desired event form, click into the :guilabel:`Tracks` smart button at the top of the
+form. If that smart button is not readily available, click the :guilabel:`More` :icon:`fa-sort-desc`
+drop-down menu to reveal hidden smart buttons.
+
+Then, click :guilabel:`Tracks` from the resulting drop-down menu.
+
+Clicking that smart button reveals the :guilabel:`Event Tracks` page for that particular event,
+which presents all the tracks (both scheduled and proposed) for the event, if there are any.
+
+.. image:: track_manage_talks/event-tracks-page.png
+   :align: center
+   :alt: Typical event tracks page for an event in the Odoo Events application.
+
+The tracks are presented in a default Kanban view, but there is also the option to view these tracks
+in a :guilabel:`List`, :guilabel:`Gantt` chart, :guilabel:`Calendar`, :guilabel:`Graph`, or
+:guilabel:`Activity` view. All of which are accessible in the upper-right corner of the
+:guilabel:`Tracks` page.
+
+In the default Kanban view, the tracks are categorized into different stages. The default stages
+are: :guilabel:`Proposal`, :guilabel:`Confirmed`, :guilabel:`Announced`, :guilabel:`Published`,
+:guilabel:`Refused`, and :guilabel:`Cancelled`. All of which can be edited, if needed.
+
+Clicking into a track from the :guilabel:`Event Tracks` page reveals the track form for that
+particular track.
+
+Create event track
+------------------
+
+From the :guilabel:`Event Tracks` page, click :guilabel:`New` to reveal a blank event track form to
+use to create an event track.
+
+.. image:: track_manage_talks/event-track-form.png
+   :align: center
+   :alt: Typical event track form in the Odoo Events application.
+
+Start by giving this track a :guilabel:`Title`.
+
+Then, if desired, add an accompanying image to the track via the :icon:`fa-pencil`
+:guilabel:`(pencil)` icon that appears when the cursor hovers over the :icon:`fa-camera`
+:guilabel:`(camera)` icon in the upper-right corner of the form. When clicked, proceed to upload the
+desired image for the track.
+
+Next, apply a :guilabel:`Track Date` for the track, and designate a :guilabel:`Location` where the
+talk is planning to be held.
+
+Then, add a :guilabel:`Duration` to the track.
+
+If the :guilabel:`Live Broadcast` setting has been enabled, users have the option to add a
+corresponding link in the :guilabel:`YouTube Video Link` field.
+
+If the :guilabel:`Always Wishlisted` checkbox is ticked, the talk is automatically set as a favorite
+for each registered attendee.
+
+Assign an internal user to be in charge of managing this event in the :guilabel:`Responsible` field.
+By default, the user who initially created the track is assigned.
```

Comment:
```suggestion
Assign an internal user to be in charge of managing this event in the :guilabel:`Responsible` field.
By default, the user who initially created the track is assigned.
```

You mention assigning a user to manage the "event" will this be management of the event or the "track"?

Reply:
Great catch. Fully my mistake. It's meant to say "track" -- not "event"